pkg/util/log/crash_reporting.go

```diff
@@ -58,21 +58,21 @@ var (
 	//
 	// Doing this, rather than just using a default of `true`, means that a node
 	// will not errantly send a report using a default before loading settings.
-	DiagnosticsReportingEnabled = settings.RegisterBoolSetting(
+	DiagnosticsReportingEnabled = settings.RegisterPublicBoolSetting(
 		"diagnostics.reporting.enabled",
 		"enable reporting diagnostic metrics to cockroach labs",
 		false,
 	)
 
 	// CrashReports wraps "diagnostics.reporting.send_crash_reports".
-	CrashReports = settings.RegisterBoolSetting(
+	CrashReports = settings.RegisterPublicBoolSetting(
```

Comment:
Since crash reporting *also* obeys the public diagnostics setting, I don't think we need to muddy the waters by also making this one public (indeed, we could probably just remove it entirely, but that's separate change)

Reply:
Done.